pyproject.toml

```diff
@@ -1,65 +1,64 @@
-[project]
-name = "skycalc_ipy"
+[tool.poetry]
+name = "skycalc-ipy"
 version = "0.3.0a0"
-# When updating the version, also
-# - update the date in anisocado/version.py
-# - update the release notese in docs/source/index.rst
-description = "Get atmospheric spectral information from the ESO skycalc server"
-readme = "README.md"
-requires-python = ">=3.8"
-# When updating the version, also update the versions in .github/workflows/*
-license = {text = "License :: OSI Approved :: GNU General Public License v3 (GPLv3)"}
-authors = [
-    {name = "Kieran Leschinski", email="kieran.leschinski@unive.ac.at"},
-]
+description = "Get atmospheric spectral information from the ESO skycalc server."
+license = "GPL-3.0-or-later"
+authors = ["Kieran Leschinski <kieran.leschinski@unive.ac.at>"]
 maintainers = [
-    {name = "Kieran Leschinski", email="kieran.leschinski@unive.ac.at"},
-    {name = "Hugo Buddelmeijer", email="hugo@buddelmeijer.nl"},
+    "Kieran Leschinski <kieran.leschinski@unive.ac.at>",
+    "Hugo Buddelmeijer <hugo@buddelmeijer.nl>",
+    "Fabian Haberhauer <fabian.haberhauer@univie.ac.at>",
```

Comment:
Note that you are not actually added as a maintainer to the wheels `METADATA` file. Instead, I'm removed. Previously:
```
Author-email: Kieran Leschinski <kieran.leschinski@unive.ac.at>
Maintainer-email: Kieran Leschinski <kieran.leschinski@unive.ac.at>, Hugo Buddelmeijer <hugo@buddelmeijer.nl>
```
new:
```
Author: Kieran Leschinski
Author-email: kieran.leschinski@unive.ac.at
Maintainer: Kieran Leschinski
Maintainer-email: kieran.leschinski@unive.ac.at
```

This apparently is [intentional because the author is not a 'multiple use' field](https://github.com/python-poetry/poetry/issues/7229). That doesn't seem like a discussion that is worthwhile to join, so I'm OK with this.

Reply:
I never quite understood the distinction between author and maintainer(s). Poetry [mentions](https://python-poetry.org/docs/pyproject/#maintainers) that it "should be distinct from authors". The maintainers listed in PyPI are taken from PyPI itself it seems, i.e. the people who have access to it there.

Anyway, as you mentioned, this is not an issue of Poetry. If we look at e.g. [ScopeSim on PyPI](https://pypi.org/project/ScopeSim/), Only the first maintainer appears, even though more than one is listed in the `METADATA`. I do realize that `METADATA` is not directly tied to PyPI and that someone might view the contents of that file another way for various reasons.

One possible solutions that come to my mind would be to just put the organisation in there. Is AstarVienna's email inbox monitored by anyone of us? To avoid the author-maintainer ambiguity, we could simply omit the optional maintainer field entirely. This seems to be the way e.g. Astropy does it. Exceptions to this might be packages that were originally developed by a single person, but then the organisation took over, in which case it would be the single maintainer.

Thoughts?

Reply:
I'd say we keep it as is for now. Let's first see what PyPI will list as authors.

If PyPI will now indeed only list Kieran, then maybe it is worthwhile to switch to the astarvienna address. I don't know whether anyone is monitors that though.

For now I just wanted to note that poetry behaves differently from setuptools, so we are at least aware of a potential problem.